common/app/views/fragments/nav/editionPicker.scala.html

```diff
@@ -0,0 +1,19 @@
+@()(implicit request: RequestHeader)
+
+@import common.{Edition, LinkTo}
+
+<div class="new-header__edition-picker">
+    @fragments.inlineSvg("world", "icon")
+
+    <select onchange="location = this.value;">
```

Comment:
Yeah I reckon that might be IE-specific. I remember I had the issue back
when I was doing a11y testing at the government: you can focus a select
field and use arrows without ever opening the menu, thereby changing the
value every time you hit the arrow key.

On Mon, 18 Jul 2016 at 11:24 Natalia Baltazar <notifications@github.com>
wrote:

> In common/app/views/fragments/nav/editionPicker.scala.html
> <https://github.com/guardian/frontend/pull/13617#discussion_r71126150>:
>
> > @@ -0,0 +1,19 @@
> > +@()(implicit request: RequestHeader)
> > +
> > +@import common.{Edition, LinkTo}
> > +
> > +<div class="new-header__edition-picker">
> > +    @fragments.inlineSvg("world", "icon")
> > +
> > +    <select onchange="location = this.value;">
>
> @regiskuckaertz <https://github.com/regiskuckaertz>
>
> This won't work via keyboard arrows (I think) because the change event is
> triggered every time the user activates an option, and in order to activate
> the third option she has to go through the first and the second.
>
> This seems to work with keyboard arrows and you have to press enter to
> activate the change event. That said, I have only tested on Chrome so far.
> I will so some cross browser testing today
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/guardian/frontend/pull/13617/files/33afa8f0509f137d067711c45a4534dbc44341dd#r71126150>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAmc2G17n6S2TCGNgWZfQr1pWbDIeppiks5qW1RJgaJpZM4JNiFe>
> .
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396